QtImageViewer/README.md

```diff
@@ -32,19 +32,31 @@ A Qt install and mesa-common-dev Debian package are required to run this example
 
     `sudo apt install mesa-common-dev`
 
+6. Follow the build instructions.
+
+7. After building the QtImageViewer, move a qt.conf file into the build file for the QtImageViewer. 
```

Comment:
We need to include what the contents of the `qt.conf` file are, and track an example of this file in a new `./res` directory. 

Here's the contents of the file:

```
[Paths]
Plugins=/usr/local/polysync/bin
```